afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/RemoteEventsBus.java

```diff
@@ -51,23 +50,19 @@ public RemoteListenableAppStorage(RemoteAppStorage storage, URI restUri) {
         }
     }
 
-    static URI getWebSocketUri(URI restUri) {
-        try {
-            String wsScheme;
-            switch (restUri.getScheme()) {
-                case "http":
-                    wsScheme = "ws";
-                    break;
-                case "https":
-                    wsScheme = "wss";
-                    break;
-                default:
-                    throw new AssertionError("Unexpected scheme " + restUri.getScheme());
-            }
-            return new URI(wsScheme, restUri.getUserInfo(), restUri.getHost(), restUri.getPort(), restUri.getPath(), restUri.getQuery(), null);
-        } catch (URISyntaxException e) {
-            throw new UncheckedUriSyntaxException(e);
-        }
+    @Override
+    public void pushEvent(NodeEvent event, String topic) {
+        topics.computeIfAbsent(topic, k -> new ArrayList<>());
+        topics.get(topic).add(event);
+    }
+
+    public Map<String, List<NodeEvent>> getTopics() {
+        return topics;
+    }
+
+    @Override
+    public void flush() {
```

Comment:
Actually, shouldn't this method push events to the server ?
For instance, I think a `BusinessEvent` pushed by a client (a user of `RemoteAppStorage`) will never reach anyone.

On the opposite, it will work for the events triggered by the server itself (storage events).

Reply:
well observed!
issue fixed